inverse.go

```diff
@@ -184,16 +184,56 @@ func (i *InverseBloomFilter) WriteTo(stream io.Writer) (int64, error) {
 // have been written by WriteTo()) from an i/o stream. It returns the number
 // of bytes read.
 func (i *InverseBloomFilter) ReadFrom(stream io.Reader) (int64, error) {
```

Comment:
There was not a test on `ReadFrom` but Alex had a test on `Encode` and `Decode` and it passes...

I'll add a test on ReadFrom to run against the old version and the new version.